Tests/FunctionalTests/TestDiscoveryTests.swift

```diff
@@ -127,4 +127,19 @@ class TestDiscoveryTests: XCTestCase {
         }
         #endif
     }
+
+    func testSubclassedTestClassTests() throws {
+        #if os(macOS)
+        try XCTSkipIf(true)
+        #endif
+        try fixture(name: "Miscellaneous/TestDiscovery/Subclass") { fixturePath in
+            let (stdout, stderr) = try executeSwiftTest(fixturePath)
+            // in "swift test" build output goes to stderr
+            XCTAssertMatch(stderr, .contains("Build complete!"))
```

Comment:
in 5.6 this was going to `stdout`

Reply:
Thanks; removed that particular check since it wasn't really important to the test.